### ISSUE TITLE
Handle Plugin Assets

### ISSUE THREAD

**Issue by robneu** [getherbert/ship#5]
It would be nice if there were some way to manage plugin assets like screenshots, an icon, and the plugin banner within the app.

**Comment by LateButEarly** [comment]
+1

**Comment by mikaelmattsson** [comment]
+1 So that you would only have to upload the retina image and it would scale it down.

**Comment by jasonagnew** [comment]
I agree, we're [talking](https://github.com/getherbert/ship/issues/9) about adding a `ship.json` where you could control this stuff.

**Comment by shivapoudel** [comment]
@jasonagnew this assets change are responsible for Admin of the plugin. If we need to place in github repo then contributors may wish to change but we need to stick with organization rules :sparkling_heart: 

**Steps guidelines:**
- Let us allow to upload the assets zip file in the ship app rather than `ship.json`
- If we upload and hit "update assets" button in ship app then update plugin assets.
- These assets update should not get shipped when finally the tag is release in github. 
- Do not wait till tag release in github and allow us to update assets whenever we wish via app.
